scripts/tests/chiptest/__init__.py

```diff
@@ -95,7 +95,6 @@ def _hardcoded_python_yaml_tests():
         "Test_TC_I_2_1.yaml",
         "Test_TC_ILL_1_1.yaml",
         "Test_TC_ILL_2_1.yaml",
-        "Test_TC_LVL_2_1.yaml",
```

Comment:
Blocking comment to give https://github.com/project-chip/connectedhomeip/pull/24629 a chance at merging first

Reply:
That PR seems to depend on flaky tests as well, so merge conflicts should not be too bad. Lets merge whichever passes CI first. I do not mind conflict resolution (should be small delta).